src/sim-host/ui/custom-elements.js

```diff
@@ -167,6 +168,7 @@ function initialize(changePanelVisibilityCallback) {
     registerCustomElement('cordova-number-entry', {
         value: {
             set: function (value) {
+                this._internalValue = value;
```

Comment:
Should this perform validation on `value` as well? Currently if the code using the `cordova-number-entry` is incorrect then it could assign a non-numeric value.